controllers/hetznerbaremetalmachine_controller.go

```diff
@@ -148,6 +150,12 @@ func (r *HetznerBareMetalMachineReconciler) Reconcile(ctx context.Context, req r
 		}
 	}()
 
+	// check whether rate limit has been reached and if so, then wait.
+	if wait := reconcileRateLimit(hbmMachine, r.RateLimitWaitTime); wait {
+		log.Info("Rate limit exceeded - requeue in 30 seconds")
```

Comment:
this log statement is redundant -> all infos are saved in conditions

Reply:
ok, I removed it there, and at the place where I copied it from.